Cargo.toml

```diff
@@ -26,7 +26,7 @@ app_dirs = "1.1.1"
 fdlimit = "0.1"
 hyper = { default-features = false, git = "https://github.com/ethcore/hyper" }
 ctrlc = { git = "https://github.com/ethcore/rust-ctrlc.git" }
-jsonrpc-core = { git = "https://github.com/ethcore/jsonrpc.git", branch = "parity-1.6" }
+jsonrpc-core = { git = "https://github.com/ethcore/jsonrpc.git", branch = "master" }
```

Comment:
I think we should use dedicated branch for referencing from parity, maybe `parity-1.7` now